test/dotnet.Tests/PackagedCommandTests.cs

```diff
@@ -65,19 +65,19 @@ public void TestProjectToolIsAvailableThroughDriver(string appName, bool useCurr
             var testInstance = TestAssets.Get(appName)
                 .CreateInstance()
                 .WithSourceFiles()
-                .WithRestoreFiles();
+                .WithNuGetConfig(new RepoDirectoriesProvider().TestPackages);
```

Comment:
They are unrelated. It's very unclear to me how this restore worked in the first place, it follows a different pattern than other tests that share the intent of restoring within the test execution directory. Restoring prior to editing the project also seemed unnecessary and potentially side-effecting. I suspect this worked previously because the package already made it into my packages directory due to the initial restore, but that implicit dependency has shown itself to be fragile in other parts of the system [LZMA tool, CrossGen] and I'd prefer to fix the symptoms by cleaning up.